src/arch/armv8/inc/arch/vgic.h

```diff
@@ -71,6 +71,9 @@ struct vgic_priv {
     struct vgic_int interrupts[GIC_CPU_PRIV];
 };
 
+#define VGIC_GICR_ACCESS    true
+#define VGIC_NO_GICR_ACCESS false
```

Comment:
Shouldnt these macros have surrounding parentheses?

```suggestion
#define VGIC_GICR_ACCESS    (true)
#define VGIC_NO_GICR_ACCESS (false)
```